dummy/src/main/java/org/ldbcouncil/finbench/impls/dummy/DummyDb.java

```diff
@@ -79,6 +83,26 @@ public static class ComplexRead1Handler implements OperationHandler<ComplexRead1
         public void executeOperation(ComplexRead1 cr1, DummyDbConnectionState dummyDbConnectionState,
                                      ResultReporter resultReporter) throws DbException {
             DummyDb.logger.info(cr1.toString());
+            List<ComplexRead1Result> complexRead1Results = new ArrayList<>();
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "101"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "102"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "103"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "104"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "105"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "106"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "107"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "108"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "109"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1010"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1011"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1012"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1013"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1014"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1015"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1016"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1017"));
+            complexRead1Results.add(new ComplexRead1Result(0, "a", "1018"));
```

Comment:
what are these lines for?

Reply:
The output of ComplexReads is the input of SimpleReads, so ComplexRead1 outputs some results for verify that SimpleReads are correct.

Reply:
Then add some comments here to note this

Reply:
Done